include/bbcode.php

```diff
@@ -586,20 +586,26 @@ function bb_ShareAttributes($share, $simplehtml)
 			}
 			break;
 		default:
-			$text = trim($share[1]) . "\n";
-
-			$avatar = proxy_url($avatar, false, PROXY_SIZE_THUMB);
-
-			$tpl = get_markup_template('shared_content.tpl');
-			$text .= replace_macros($tpl, array(
-					'$profile' => $profile,
-					'$avatar' => $avatar,
-					'$author' => $author,
-					'$link' => $link,
-					'$posted' => $posted,
-					'$content' => trim($share[3])
-				)
-			);
+			// Transforms quoted tweets in rich attachments to avoid nested tweetsx
+			if (stripos(normalise_link($link), 'http://twitter.com/') === 0) {
```

Comment:
What will happen when rich oembedding is deactivated?

Reply:
That's a very good question, I'll test it.